BREAKING_CHANGE_AND_DEPRECATION.md

```diff
@@ -0,0 +1,46 @@
+# Breaking changes and deprecations
+
+Breaking changes are defined as a change to any of the following that causes installation errors or 
+unexpected runtime behavior after upgrading to the next stable minor version of Ratify(cli, plugin,):
+- Verification result and schema
+- Default configuration value
+- User facing plugin interfaces
+
+We do our best to maintain backwards compatibility, however breaking changes may be introduced between version for features that have not yet reached v1.0.0 or only available in RATIFY_EXPERIMENTAL flag.
+
+### Features in Preview
```

Comment:
do we only focus on breaking features or do we consider all new features?

Reply:
Features in EXPERIMENTAL , vs feature in beta